module/VuFind/src/VuFind/Db/Entity/AccessToken.php

```diff
@@ -66,40 +66,40 @@ class AccessToken implements AccessTokenEntityInterface
     #[ORM\Column(name: 'type', type: 'string', length: 128, nullable: false)]
     #[ORM\Id]
     #[ORM\GeneratedValue(strategy: 'NONE')]
-    protected $type;
+    protected string $type;
 
     /**
      * User.
      *
      * @var UserEntityInterface
      */
     #[ORM\JoinColumn(name: 'user_id', referencedColumnName: 'id')]
-    #[ORM\ManyToOne(targetEntity: \VuFind\Db\Entity\User::class)]
-    protected $user;
+    #[ORM\ManyToOne(targetEntity: \VuFind\Db\Entity\UserEntityInterface::class)]
```

Comment:
Do we need the fully qualified name here?

```suggestion
    #[ORM\ManyToOne(targetEntity: UserEntityInterface::class)]
```

Reply:
I don't think we do!